test/beanService.test.ts

```diff
@@ -0,0 +1,57 @@
+import { jest, describe, expect, test } from "@jest/globals";
+import prisma from "../src/config/prisma";
+import CreateBeanInput from "../src/interfaces/createBeanInput";
+import BeanService from "../src/services/beanService";
+
+const b1: CreateBeanInput = {
+    brand: "Lavazza",
+    model: "Gold",
+    roast: "Light",
+};
+
+jest.setTimeout(10000);
```

Comment:
Do we need this timeout, I specifically added it to a test because it was timing out. Try testing without it and see if it passes without timing out.

Reply:
You are right, the timeout was not needed